### PR TITLE
[zh] sync 1.27 extensible admission controllers

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
[zh] sync 1.27 extensible-admission-controllers

### CODE REVIEW

File: content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md [hunk 1]
```diff
@@ -1059,6 +1066,100 @@ The `matchPolicy` for an admission webhooks defaults to `Equivalent`.
...
+### 匹配请求：`matchConditions`  {#matching-requests-matchConditions}
+
+{{< feature-state state="alpha" for_k8s_version="v1.27" >}}
+
+{{< note >}}
+<!--
+Use of `matchConditions` requires the [featuregate](/docs/reference/command-line-tools-reference/feature-gates/)
+`AdmissionWebhookMatchConditions` to be explicitly enabled on the kube-apiserver before this feature can be used.
+-->
+使用 `matchConditions` 需要先在 kube-apiserver 上明确启用[功能门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md:1081]:
```suggestion
使用 `matchConditions` 需要先在 kube-apiserver
上明确启用[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
```

File: content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md [hunk 2]
```diff
@@ -1059,6 +1066,100 @@ The `matchPolicy` for an admission webhooks defaults to `Equivalent`.
...
+{{< /note >}}
+
+<!--
+You can define _match conditions_for webhooks if you need fine-grained request filtering. These
+conditions are useful if you find that match rules, `objectSelectors` and `namespaceSelectors` still
+doesn't provide the filtering you want over when to call out over HTTP. Match conditions are
+[CEL expressions](/docs/reference/using-api/cel/). All match conditions must evaluate to true for the
+webhook to be called.
+-->
+如果你需要细粒度地过滤请求，你可以为 Webhook 定义 _match conditions_。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md:1092]:
```suggestion
如果你需要细粒度地过滤请求，你可以为 Webhook 定义 **匹配条件**。
```

File: content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md [hunk 3]
```diff
@@ -1059,6 +1066,100 @@ The `matchPolicy` for an admission webhooks defaults to `Equivalent`.
...
+Match conditions have access to the following CEL variables:
+-->
+匹配条件可以访问以下 CEL 变量：
+
+<!--
+- `object` - The object from the incoming request. The value is null for DELETE requests. The object
+  version may be converted based on the [matchPolicy](#matching-requests-matchpolicy).
+-->
+- `object` - 来自传入请求的对象。对于 DELETE 请求，该值为 null。根据 [matchPolicy]((#matching-requests-matchpolicy))
+  可能会转换对象版本。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md:1115]:
```suggestion
- `object` - 来自传入请求的对象。对于 DELETE 请求，该值为 null。
  根据 [matchPolicy]((#matching-requests-matchpolicy)) 可能会转换对象版本。
```

File: content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md [hunk 4]
```diff
@@ -1059,6 +1066,100 @@ The `matchPolicy` for an admission webhooks defaults to `Equivalent`.
...
+-->
+- `request` - [AdmissionReview](#request) 的请求部分，不包括 object 和 oldObject。
+<!--
+- `authorizer` - A CEL Authorizer. May be used to perform authorization checks for the principal
+  (authenticated user) of the request. See
+  [Authz](https://pkg.go.dev/k8s.io/apiserver/pkg/cel/library#Authz) in the Kubernetes CEL library
+  documentation for more details.
+-->
+- `authorizer` - 一个 CEL 授权器。可用于对请求的主体（经过身份认证的用户）执行授权检查。
+  更多详细信息，请参阅 Kubernetes CEL 库文档中的 [Authz](https://pkg.go.dev/k8s.io/apiserver/pkg/cel/library#Authz)。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md:1131]:
```suggestion
  更多详细信息，请参阅 Kubernetes CEL 库文档中的
  [Authz](https://pkg.go.dev/k8s.io/apiserver/pkg/cel/library#Authz)。
```

File: content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md [hunk 5]
```diff
@@ -1059,6 +1066,100 @@ The `matchPolicy` for an admission webhooks defaults to `Equivalent`.
...
+- `authorizer.requestResource` - A shortcut for an authorization check configured with the request
+  resource (group, resource, (subresource), namespace, name).
+-->
+- `authorizer.requestResource` - 对配置的请求资源（组、资源、（子资源）、名字空间、名称）进行授权检查的快捷方式。
+
+<!--
+For more information on CEL expressions, refer to the
+[Common Expression Language in Kubernetes reference](/docs/reference/using-api/cel/).
+-->
+了解有关 CEL 表达式的更多信息，请参阅 [Kubernetes 参考文档中的通用表达式语言](/zh-cn/docs/reference/using-api/cel/)。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md:1142]:
```suggestion
了解有关 CEL 表达式的更多信息，请参阅
[Kubernetes 参考文档中的通用表达式语言](/zh-cn/docs/reference/using-api/cel/)。
```

**Review by windsonsea — COMMENTED** [review]
en 做了折行处理，那么每一行 zh 尽量不要超过 en 的长度

File: content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md [hunk 6]
```diff
@@ -1059,6 +1066,103 @@ The `matchPolicy` for an admission webhooks defaults to `Equivalent`.
...
+- `request` - The request portion of the [AdmissionReview](#request), excluding `object` and `oldObject`.
+-->
+- `request` - [AdmissionReview](#request) 的请求部分，不包括 object 和 oldObject。
+<!--
+- `authorizer` - A CEL Authorizer. May be used to perform authorization checks for the principal
+  (authenticated user) of the request. See
+  [Authz](https://pkg.go.dev/k8s.io/apiserver/pkg/cel/library#Authz) in the Kubernetes CEL library
+  documentation for more details.
+-->
+- `authorizer` - 一个 CEL 授权器。可用于对请求的主体（经过身份认证的用户）执行授权检查。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md:1131]:
```suggestion
- `authorizer` - 一个 CEL 鉴权组件。可用于对请求的主体（经过身份认证的用户）执行鉴权检查。
```

File: content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md [hunk 7]
```diff
@@ -1059,6 +1066,103 @@ The `matchPolicy` for an admission webhooks defaults to `Equivalent`.
...
+[Common Expression Language in Kubernetes reference](/docs/reference/using-api/cel/).
+-->
+了解有关 CEL 表达式的更多信息，请参阅
+[Kubernetes 参考文档中的通用表达式语言](/zh-cn/docs/reference/using-api/cel/)。
+
+<!--
+In the event of an error evaluating a match condition the webhook is never called. Whether to reject
+the request is determined as follows:
+-->
+如果在求值匹配条件时出现错误，则不会调用 Webhook。根据以下方式确定是否拒绝请求：
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/extensible-admission-controllers.md:1151]:
```suggestion
如果在对匹配条件求值时出现错误，则不会调用 Webhook。根据以下方式确定是否拒绝请求：
```

**Review by tengqm — COMMENTED** [review]
Hold on, we are moving this YAML back into inline.

**Pull request merged**